docs/libertyPackage.md

```diff
@@ -2,13 +2,15 @@
 
 Package a Liberty server.
 
+Starting with WebSphere Liberty 8.5.5.9, it is possible to package a server into an executable jar file by setting the include parameter to runnable. The created jar file can be executed using the java -jar command.
+
 ### Properties
 
 | Attribute | Description | Required |
 | --------- | ------------ | ----------|
-| include | A comma-delimited list of values. The valid values vary depending on the task. For the libertyPackage task the valid values are all, usr, and minify and must be declared in the packageLiberty closure. | Yes, only when the `os` option is set. |
-| archive | Location of the target archive file. Used with the libertyPackage task on its respective closure. | No |
-| os| A comma-delimited list of operating systems that you want the packaged server to support. Used in the packageLiberty closure. The 'include' attribute __must__ be set to `minify`. | No |
+| include | Packaging type. Can be used with values `all`, `usr`, `minify`, `wlp`, `runnable`, `all,runnable` and `minify,runnable`. The default value is `all`. The `runnable`, `all,runnable` and `minify,runnable` values are supported beginning with 8.5.5.9 and works with `jar` type archives only. This must be declared in the `packageLiberty` closure. | Yes, only when the `os` option is set. |
+| archive | Location of the target file or directory. If the 'archive' is set to an existing directory, the contents of the server instance will be compressed into `${archive}/${project.name}.zip`&#124;`jar` file. If the target file or directory doesn't exist, the contents of the server instance will be compressed into the specified file, `${buildDir}/${archive}.zip`&#124;`jar`. If the `archive` option is not set, it defaults to `${buildDir}/libs/${project.name}.zip`&#124;`jar`. A jar file is created when the packaging type is either `runnable`,`all,runnable` or `minify,runnable`. A zip file is created for other packaging types. | No |
```

Comment:
I think it is incorrect to say:

`If the target file or directory doesn't exist, the contents of the server instance will be compressed into the specified file, ${buildDir}/${archive}.zip&#124;jar.`

But, you can simply change it to:

`If the target file or directory doesn't exist, the contents of the server instance will be compressed into the specified file.`